pusher/src/Controller/MapController.ts

```diff
@@ -107,6 +107,8 @@ export class MapController extends BaseHttpController {
                 return;
             }
 
+            adminApi.setLocale(req.header("accept-language"));
```

Comment:
Same remark here. Let's get rid of `setLocale` and let's explicitly pass the language in parameter of the adminApi methods.